package.json

```diff
@@ -17,7 +17,7 @@
     "lint": "eslint bin lib test examples client-src",
     "test": "nyc --reporter lcovonly mocha --full-trace --check-leaks --exit",
     "prepare": "npm run -s prepare:delete-certs && npm run -s transpile:index && npm run -s build:live && npm run -s build:index && npm run -s build:sockjs",
-    "prepare:delete-certs": "node lib/delete-certs.js",
+    "prepare:delete-certs": "./node_modules/.bin/rimraf ./ssl/*.pem",
```

Comment:
you can avoid `./node_modules/.bin/`

Reply:
@michael-ciniawsky beat you to the punch. Already fixed :)